src/Std/Data/DTreeMap/Basic.lean

```diff
@@ -218,27 +234,43 @@ Tries to retrieve the mapping for the given key, returning `none` if no such map
 def get? (t : DTreeMap α β cmp) (a : α) : Option β :=
   letI : Ord α := ⟨cmp⟩; Impl.Const.get? a t.inner
 
+@[inline, inherit_doc get?, deprecated get? (since := "2025-02-12")]
+def find? (t : DTreeMap α β cmp) (a : α) : Option β :=
+  get? t a
+
 /--
 Given a proof that a mapping for the given key is present, retrieves the mapping for the given key.
 -/
 @[inline]
 def get (t : DTreeMap α β cmp) (a : α) (h : a ∈ t) : β :=
   letI : Ord α := ⟨cmp⟩; Impl.Const.get a t.inner h
 
+@[inline, inherit_doc get, deprecated get (since := "2025-02-12")]
+def find (t : DTreeMap α β cmp) (a : α) (h : a ∈ t) : β :=
```

Comment:
I don't think `Lean.RBMap.find` exists

Reply:
It does exist, although in line 205 and not grouped with the others.

Reply:
I see. It's a different function, though, as it behaves more like `find?`.